timesketch/frontend-ng/src/components/Explore/EventDetail.vue

```diff
@@ -24,17 +24,30 @@ limitations under the License.
                 <tr v-for="(value, key) in fullEventFiltered" :key="key">
                   <td>
                     {{ key }}
+                  </td>
+                  <td v-if="key.includes('xml') || checkContextLinkDisplay(key, value)" class="px-0">
```

Comment:
Consider moving this into the menu instead of rendering its own icon. Not necessary in the PR, but maybe add a TODO?

Reply:
I thought about that earlier as well. However, I'm not sure how this will work with the expectations created by the icon type (link-extern). 
I opened #2449 to research and track this topic.